src/main/java/de/metas/ui/web/window/datatypes/json/DateTimeConverters.java

```diff
@@ -232,10 +243,18 @@ else if (valueObj instanceof CharSequence)
 			{
 				return null;
 			}
-			else if (isPossibleJdbcTimestamp(valueObj))
+			if (isPossibleJdbcTimestamp(json))
 			{
-				final Timestamp timestamp = fromJdbcTimestampToLocalDateTime(valueObj);
-				return fromObjectConverter.apply(timestamp);
+				try
+				{
+					final Timestamp timestamp = fromPossibleJdbcTimestamp(json);
+					return fromObjectConverter.apply(timestamp);
+				}
+				catch (final Exception e)
+				{
+					logger.warn("Error while converting possible JDBC Timestamp String to Timestamp. String to parse: {}. Exception: {}", json, e);
```

Comment:
```suggestion
					logger.warn("Error while converting possible JDBC Timestamp `{}` to java.sql.Timestamp", json, e);
```

Basically:
* u don't have to explicitly have a placeholder for your exception (i.e. Exception: {}). Just have the exception as last param and logback will handle it (check `ch.qos.logback.classic.spi.LoggingEvent.extractThrowableAnRearrangeArguments(Object[])` for reference)
*  I've shorten ur message a bit (that's opinioned, based on my taste)